backend/src/main/java/com/redhat/cloud/notifications/auth/kessel/KesselAuthorization.java

```diff
@@ -113,7 +109,7 @@ public void hasPermissionOnResource(final SecurityContext securityContext, final
         }
 
         // Stop the timer.
-        permissionCheckTimer.stop(this.meterRegistry.timer(KESSEL_METRICS_PERMISSION_CHECK_TIMER_NAME, Tags.of(KESSEL_METRICS_TAG_PERMISSION_KEY, permission.getKesselPermissionName(), KESSEL_METRICS_TAG_RESOURCE_TYPE_KEY, resourceType.getKesselName())));
+        permissionCheckTimer.stop(this.meterRegistry.timer(KESSEL_METRICS_PERMISSION_CHECK_TIMER_NAME, Tags.of(KESSEL_METRICS_TAG_PERMISSION_KEY, permission.getKesselPermissionName(), Constants.KESSEL_METRICS_TAG_RESOURCE_TYPE_KEY, resourceType.getKesselName())));
```

Comment:
Would you be interested in a client that integrated with micrometer out of the box? (so you didn't have to do your own wrapping timers all overal?)

Reply:
As stated in the meeting, it would be nice as long as it is configurable, but I don't think we mind having to implement these calls ourselves either. So for now let's keep them and we'll come back to them.